src/components/commons/tag/Tag.tsx

```diff
@@ -0,0 +1,40 @@
+'use client';
+
+import classNames from 'classnames';
+
+type TagProps = {
+  text: string;
+  color: string;
+  size: 'lg' | 'sm';
+  round?: boolean;
+};
+
+export default function Tag({ text, color, round, size }: TagProps) {
+  const roundClass = round ? 'rounded-full' : 'rounded';
+  const colorVariants: { [key: string]: string } = {
+    toss: 'bg-toss-blue-light text-toss-blue',
+    blue: 'bg-blue-light-chip text-blue-chip',
+    green: 'bg-green-light-chip text-green-chip',
+    orange: 'bg-orange-light-chip text-purple-chip',
+    pink: 'bg-pink-light-chip text-pink-chip',
+    red: 'bg-red-light-chip text-red-chip',
+    purple: 'bg-purple-light-chip text-purple-chip',
+  };
+  const sizeClass = classNames({
+    'text-12': size === 'lg',
+    'text-10': size === 'sm',
+  });
+
+  return (
+    <span
+      className={`text-12 inline-flex items-center rounded px-8 py-4 ${colorVariants[color]} ${roundClass} ${sizeClass}`}
+    >
+      {round && (
+        <span
+          className={`mr-6 inline-block h-6 w-6 rounded-full bg-toss-blue`}
+        ></span>
+      )}
+      <span>{text}</span>
```

Comment:
재사용 측면에서 prop으로 text 대신 children을 받는 것은 어떨까요?

```suggestion
      {children}
```

만약 변경하신다면 Tag를 사용할 때는 이렇게 쓸 수 있을 것 같습니다!

```tsx
      <Tag color="pink" size="sm">
        프로젝트
      </Tag>

      <Tag color="toss" size="lg" round={true}>
        <span className="flex items-center gap-6">
          <ColorCircle size="sm" color="bg-toss-blue" />
           To do
        </span>
      </Tag>
```

Reply:
재사용 측면에선 children이 더 낫겠네요! 감사합니다!